polyglot/piranha/src/lib.rs

```diff
@@ -93,11 +96,30 @@ pub fn execute_piranha(
   }
 
   // flag_cleaner.relevant_files
-  flag_cleaner
+  let summaries = flag_cleaner
     .get_updated_files()
     .iter()
     .map(PiranhaOutputSummary::new)
-    .collect_vec()
+    .collect_vec();
+  log_piranha_output_summaries(&summaries);
+  return summaries;
+}
+
+fn log_piranha_output_summaries(summaries: &Vec<PiranhaOutputSummary>) {
+    let mut total_number_of_matches : usize = 0;
+    let mut total_number_of_rewrites : usize = 0;
+    for summary in summaries {
+      let number_of_rewrites =&summary.rewrites().len();
+      let number_of_matches = &summary.matches().len();
+      info!("File : {:?}", &summary.path());
+      info!("  {}", format!("# Rewrites : {number_of_rewrites}"));
```

Comment:
This (and others below) still seem like unnecessary calls to `format!`, unless I am missing something. Also, wouldn't it be better to indent with tabs? (i.e. `info!("\t# Rewrites : {number_of_rewrites}");` )

Reply:
Yea. Thats the thing. `info!` does not support formatting strings using values of variables from the context. That is something that only the `format!` marcro supports. 
If we had to do it using info I would do something like: 
`info!(" # Rewrites : {}", number_of_rewrites))`
I will update it to this pattern (in cases where I am not using `color`)